portal-backend/depmap/settings/settings.py

```diff
@@ -101,7 +101,16 @@ def flagship_projects(self):
 
     @property
     def new_compound_page_tabs(self):
-        return self.is_skyros()
+        return self.is_prerelease_env()
+
+    # TODO: This should be VERY temporary and is only here to hide
+    # the extra datasets while discrepancies in IDs are resolved.
+    # This only affects the dose curves tab that pulls from the legacy db
+    # for the plot due to a reliance on old sqlite tables to draw
+    # the curves and replicates.
+    @property
+    def show_all_new_dose_curve_tab_datasets(self):
+        return False
```

Comment:
Excellent! I like that you put this behind a feature flag, and I appreciate the extra comments about why it was added. Hopefully that will make it easy to update this once the reprocessing changes are done.